### DISCUSSION TITLE
Model2 parameter settings

### DISCUSSION THREAD

**Discussion by christyny** [LAPKB/Pmetrics#65] [Q&A] [answered]
Hello,when I run the following model parameters, the run reports an error.

Model parameters:
'#Primary
Ke, 0.01,0.2
KCP, 0.01,0.2
KPC, 0.01,0.2
V0, 5,50
Vp, 5,100

#Cov
WT
ASIAN
AGE
GENDER

#Sec
V=V0*(WT/56)^(0.75)
CL=Ke*V0
C 56 is the mean weight in the population

#Out
Y(1) = X(1)/V

#Err
G=5
0.02, 0.05, -0.0002, 0

Code to run
'> setwd(paste(wd,"/Runs",sep=""))
> file.copy(from=c("../src/model2.txt","../src/ex.csv"),to=getwd(),overwrite=T)
[1] TRUE TRUE
> NPrun(data="ex.csv",model="model2.txt")
Error: 
You have Fortran syntax errors in your model statments, as detailed above.
In addition: Warning message:
In system(cmd, intern = intern, wait = wait | intern, show.output.on.console = wait,  :
  running command 'C:\windows\system32\cmd.exe /c gfortran -fsyntax-only model2.for' had status 1

**Comment by Trepon** [comment, marked as answer]
Hello,
`V=V0*(WT/56)^(0.75)` should be replaced with `V=V0*(WT/56)**0.75`.
The exponentiation operator should be `**`, not `^`.
Good luck!

**Reply by christyny** [reply]
Great thanks for your help!
